android/app/src/main/java/com/google/codeu/chatme/view/MainActivity.java

```diff
@@ -14,6 +14,6 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // TODO: implement basic splash screen in this activity
-        startActivity(new Intent(this, LoginActivity.class));
+        startActivity(new Intent(this, SplashScreen.class));
```

Comment:
I think you can just delete `MainActivity` and its layout now. It was just a placeholder for the `SplashScreen` activity you created.